### ISSUE TITLE
Train the Model

### ISSUE THREAD

**Issue by Zach-Stealth-Wilson** [Thomas-J-Kidd/Final-Project-3#7]
Set up the training environment 

Train the model using the training dataset while validating its performance on the validation set.

Tune hyperparameters to optimize performance.

Regularly save and document the model versions and their performance metrics.
